scanpy/settings.py

```diff
@@ -1,7 +1,7 @@
 """Settings
 """
 
-verbosity = 'warning'
+verbosity = 'warn'
```

Comment:
This is strange, are you sure this complies with what we have? This might break the whole logging for all Scanpy users...

Reply:
In:

`_VERBOSITY_LEVELS_FROM_STRINGS = {
    'error': 0,
    'warn': 1,
    'info': 2,
    'hint': 3,
}
`

There is no key `warning`, and this returned `KeyError` error

Reply:
Thanks, @awnimo. @flying-sheep, did you change this but you didn't change the default of the global `settings` module? That sounds strange.